server/core/workspace/workspace_create.go

```diff
@@ -0,0 +1,54 @@
+package workspace
+
+import (
+	"context"
+	"fmt"
+
+	workspacesv1alpha1 "github.com/konflux-workspaces/workspaces/operator/api/v1alpha1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+//go:generate mockgen -destination=mocks/mocks.go -package=mocks . WorkspaceCreater
+
+// CreateWorkspaceCommand contains the information needed to create a new workspace
+type CreateWorkspaceCommand struct {
+	Workspace workspacesv1alpha1.Workspace
+}
+
+// CreateWorkspaceResponse contains the newly-created workspace
+type CreateWorkspaceResponse struct {
+	Workspace *workspacesv1alpha1.Workspace
+}
+
+type WorkspaceCreater interface {
```

Comment:
Shouldn't it be `creator`?

```suggestion
type WorkspaceCreator interface {
```

Reply:
Fixed in 212c7a1.